packages/terra-table/src/subcomponents/Row.jsx

```diff
@@ -81,6 +86,7 @@ function Row(props) {
     hasRowSelection,
     id,
     isSelected,
+    isTableStriped,
```

Comment:
nit: Like my other comment, could we just name this `isStriped`? The naming feels weird compared to the other variable names.

Reply:
This naming convention is similar.  The row may or may not be striped.  It is the parent table that has striping enabled.  We have CSS to control this at the row component level though.